ingest/cell_metadata.py

```diff
@@ -182,11 +182,12 @@ def validate_header_keyword(self):
         if self.headers[0].casefold() == 'NAME'.casefold():
             valid = True
             if self.headers[0] != 'NAME':
-                # ToDO - capture warning below in issue report
-                print(
+                msg = (
                     f'Warning: metadata file keyword NAME provided as '
                     f'{self.headers[0]}'
                 )
+                self.store_validation_issue('warn', 'format', msg, '')
```

Comment:
Given the method signature:

`def store_validation_issue(self, type, category, msg, associated_info=None)`

and looking at the method, it seems

`self.store_validation_issue('warn', 'format', msg, '')`

and

`self.store_validation_issue('warn', 'format', msg)`

would result in the same ultimate output.  So this seems simpler:

```suggestion
                self.store_validation_issue('warn', 'format', msg)
```

And similarly for the other cases of passing `''` in this parameter.

Reply:
Thanks for catching that - had trouble getting store_validation_issue to work at one point and I think that was my chasing down a red herring...

Reply:
updated in `4023a08`